dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: OpenAMP (RPMsg with static VRINGs) backend
+
+compatible: "zephyr,ipc-openamp-static-vrings"
```

Comment:
This compatible doesn't match the one in the commit message

Reply:
Oooh right, it was changed `s/subsys/zephyr`. I'll fix that.